clients/src/test/java/org/apache/kafka/clients/consumer/internals/FetcherTest.java

```diff
@@ -593,6 +593,53 @@ public void testFetchOffsetOutOfRangeException() {
     }
 
     @Test
+    public void testFetchPositionAfterException() {
```

Comment:
Can we add a unit test about subscription change / offset seek after an exception is cached?